src/Expensify.js

```diff
@@ -181,7 +180,7 @@ function Expensify(props) {
     }
 
     return (
-        <DeeplinkWrapper>
+        <>
```

Comment:
> This will cause regression of not showing open desktop app popup on unauthorized screens (i.e. login page, public chat room)

This is intended. We asked in Slack and there was no indication that the prompt to open the app in desktop prior to sign-in was essential: https://expensify.slack.com/archives/C01GTK53T8Q/p1683070404415199

Sorry that wasn't conveyed more clearly.

Reply:
Follow up from Slack thread: It looks like this should still work, because `authToken` is still set even in this "anonymous user" use case (https://expensify.slack.com/archives/C01GTK53T8Q/p1686181191875739?thread_ts=1683070404.415199&cid=C01GTK53T8Q)